src/services/user/adminBlockUser.js

```diff
@@ -1,12 +1,18 @@
 const {successResponse, errorResponse} = require('../../utils');
 const {adminBlockUserProcess} = require('../../processes/scoring/admin-block-unblock-user');
+const UsersFunction = require('../../databases/functions/users');
 
 const adminBlockUser = async (req, res) => {
   try {
     const {userId} = req.body;
     if (!userId) {
       return errorResponse(res, 'userId is required', 400);
     }
+    try {
+      await UsersFunction.userBlockByAdmin(userId);
+    } catch (error) {
+      return errorResponse(res, 'Failed to block user by admin', 500);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling for `UsersFunction.userBlockByAdmin(userId)` is good, but it would be better if you could log the actual error message. This will help in debugging if any issues arise.
```diff
-    } catch (error) {
-      return errorResponse(res, 'Failed to block user by admin', 500);
+    } catch (error) {
+      console.error(error);
+      return errorResponse(res, `Failed to block user by admin: ${error.message}`, 500);
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->